src/pages/home/report/ReportActionsView.js

```diff
@@ -558,4 +590,12 @@ export default compose(
     withWindowDimensions,
     withDrawerState,
     withLocalize,
+    withOnyx({
+        personalDetails: {
+            key: ONYXKEYS.PERSONAL_DETAILS,
+        },
+        policies: {
```

Comment:
Is it used?

Reply:
personalDetails  yes to use chat users but no need for policies (removed)